src/views/queries.tsx

```diff
@@ -10,10 +10,13 @@ export default class QueriesTemplates {
       <Page
         title="Veritable - Queries"
         heading="Queries"
-        headerLinks={[{ name: 'Queries', url: '/queries' }]}
+        headerLinks={[
+          { name: 'Query Management', url: '/query-management' },
+          { name: 'Queries', url: '/query-management/queries' },
+        ]}
         stylesheets={['new-invite.css']}
```

Comment:
We shouldb't have the `new-invite` stylesheet in this page. I should have spotted this in the previous PR sorry